tests/expectations/body.cpp

```diff
@@ -47,8 +53,53 @@ union MyUnion {
   int32_t extra_member; // yolo
 };
 
+struct MyFancyStruct_Prepended {
+#ifdef __cplusplus
+  inline void prepended_wohoo();
+#endif
+  int32_t i;
+};
+
+struct MyFancyEnum_Prepended {
+  enum class Tag {
+    Foo_Prepended,
+    Bar_Prepended,
+    Baz_Prepended,
+  };
+
+  struct Bar_Prepended_Body {
+    int32_t _0;
+  };
+
+  struct Baz_Prepended_Body {
+    int32_t _0;
+  };
+
+  #ifdef __cplusplus
+    inline void wohoo();
```

Comment:
May be worth to put this at the top of everything? That allows to make everything private which can be nice. wdyt?

Reply:
Whoops. I most have overlooked this.